### ISSUE TITLE
Change Reposity Name To `forecasttools-R`?

### ISSUE THREAD

**Issue by AFg6K7h4fhy2** [CDCgov/forecasttools#11]
From previous conversations with DHM, the author recalls the plan to have a "forecasttools-R" and a "forecasttools-py" (see [here for latter repo.](https://github.com/CDCgov/forecasttools-py)). If this plan is no longer active, then this issue can be closed. If the change in name of repository is still pending, this issue serves as a reminder.

**Comment by dylanhmorris** [comment]
Some R tooling assumes package name = repo name, so I think it's simplest to keep this as `forecasttools` for now.

**Comment by AFg6K7h4fhy2** [comment]
> Some R tooling assumes package name = repo name. While things can be configured to get around this, I think it's simplest to keep this as `forecasttools` for now.

Agree. My bad for not closing this issue sooner, I had forgotten about it.
